boilerplate/App/Config/DebugConfig.js

```diff
@@ -4,5 +4,7 @@ export default {
   yellowBox: __DEV__,
   reduxLogging: __DEV__,
   includeExamples: __DEV__,
-  useReactotron: __DEV__
+  useReactotron: __DEV__,
+  launchStorybook: __DEV__
```

Comment:
I would set the default to "false" because otherwise it will launch storybook entrypoint everytime